### ISSUE TITLE
[Spells/Blood Queen] DK Anti-magic Shell can dispell Swarming Shadows 

### ISSUE THREAD

**Issue by Keader** [WoWManiaUK/Redemption#4460]
Fixed a bug that allow Death Knights dispel Swarming Shadows, if use Anti Magic shell in first 2 seconds.

Correct Behavior: Players should still get fire, even use anti magic shell

**Comment by svijetlo** [comment]
was this only for AMS of DK or for rest of that kind of spells 
as cloak of shadows (rogue) and divine shield (paladin) should be able to remove it

**Comment by Keader** [comment]
Spells that DISPEL magic effects, can remove those spells.
It means: Ice Block, Divine Shield, can remove it.
AMS is not a dispel effect, so... should not remove it.

**Comment by Aristo14** [comment]
Regression, I was able to dispell the effect with AMS yesterday in raid.

**Comment by Keader** [comment]
I found a new issue on core that conflicts with everything.
Test now, should be good.

**Comment by svijetlo** [comment]
good as it get 20 out of 20 was not blocked by AMS you just poo purple flames
